Sensus.Shared.iOS/Callbacks/UILocalNotifications/UILocalNotificationNotifier.cs

```diff
@@ -62,7 +48,7 @@ public override void IssueNotificationAsync(string message, string id, bool play
                         AlertBody = message,
                         TimeZone = null,  // null for UTC interpretation of FireDate
                         FireDate = DateTime.UtcNow.ToNSDate(),
-                        UserInfo = new NSDictionary(NOTIFICATION_ID_KEY, id)
```

Comment:
See my comment above. The conflation of NOTIFICATION_ID_KEY with SENSUS_CALLBACK_ID_KEY seemed to exist before our refactor? Where I felt I knew how to address I did. In other places where it appeared things didn't work as intended to me I removed. You certainly know the intent of these two id's better than me.